src/integrationTest/java/uk/gov/hmcts/reform/bulkscanning/features/SasTest.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.bulkscanning.features;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.palantir.docker.compose.DockerComposeRule;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.net.URISyntaxException;
+import java.security.InvalidKeyException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SasTest {
+
+    public static final String storageConnectionString = "UseDevelopmentStorage=true";
+
+    @ClassRule
+    public static DockerComposeRule docker = DockerComposeRule.builder()
+        .file("src/integrationTest/resources/docker-compose.yml")
+        .build();
+    CloudBlobContainer root;
```

Comment:
this looks super weird at first glance :)
Perhaps add an empty line (and private/protected?)